shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-postgresql/src/main/java/org/apache/shardingsphere/proxy/frontend/postgresql/auth/PostgreSQLAuthenticationHandler.java

```diff
@@ -66,7 +66,7 @@ public static PostgreSQLLoginResult loginWithMd5Password(final String username,
         return new PostgreSQLLoginResult(PostgreSQLErrorCode.SUCCESSFUL_COMPLETION, null);
     }
     
-    private static String md5Encode(final String username, final String password, final byte[] md5Salt) {
+    static String md5Encode(final String username, final String password, final byte[] md5Salt) {
```

Comment:
why remove private?

Reply:
It's called in `PostgreSQLAuthenticationEngineTest`, since `md5Salt` is generated random bytes in `PostgreSQLAuthenticationEngine`, test case will invoke `md5Encode` to calculate `md5Digest` for login.

Reply:
Test case will use Java reflection to get it, just like `MySQLAuthenticationEngineTest`.